src/libraries/Microsoft.PowerFx.Core/Utils/DPath.cs

```diff
@@ -179,67 +180,14 @@ public DPath Append(DPath path)
             return new DPath(node);
         }
 
-        public DPath GoUp(int count)
```

Comment:
> public DPath GoUp(int count) [](http://example.com/codeflow?start=7&length=29)

Are we removing this? It looks like it's still used.

Reply:
Yup, removed. Thanks, I missed that during testing, the PA Client build hid the errors.